aea/protocols/base.py

```diff
@@ -297,36 +265,6 @@ def decode(obj: bytes) -> Message:
         return msg
 
 
-class JSONSerializer(Serializer):
```

Comment:
We cannot maintain this Serializer as it does not preserve all structure on the objects passed in `body`. It is also not used, so can be safely removed.

Reply:
Agree